wagtail/models/reference_index.py

```diff
@@ -456,6 +456,14 @@ def create_or_update_for_object(cls, object):
         # Construct the set of reference records that have been found on the object but are not
         # already present in the database
         new_references = references - set(existing_references.keys())
+
+        # ignore_conflicts is not available on MSSQL server
+        # refs https://github.com/wagtail/wagtail/pull/11025#issuecomment-1755253875
+        if connection.vendor == "microsoft":
```

Comment:
Let's use Django's `DatabaseFeatures`, this is how they and third party packages tend to check for feature support:

https://github.com/django/django/blob/e083f3082c71853a01bf149bda7fdbaf58d25f4d/django/db/backends/base/features.py#L293-L295

```suggestion
        if connection.features.supports_ignore_conflicts:
```

Reply:
ooh nice. TIL :mortar_board: